tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/CatalogUtil.java

```diff
@@ -552,6 +552,16 @@ public static AlterTableDesc setProperty(String tableName, KeyValueSet params, A
     return alterTableDesc;
   }
 
+  public static AlterTableDesc unsetProperty(String tableName, String[] propertyKeys, AlterTableType alterTableType) {
```

Comment:
Right. All those methods need to be refined. But, IMO, they are all internal APIs, and don't have to be consistent. So, I think that the new method has a better signature.